src/BuildScriptGenerator/BaseBashBuildScript.sh.tpl

```diff
@@ -31,7 +31,10 @@ then
 	then
 		echo
 		echo "Destination directory is not empty. Deleting its contents..."
+		START_TIME=$SECONDS
 		rm -rf "$DESTINATION_DIR"/*
+		ELAPSED_TIME=$(($SECONDS - $START_TIME))
```

Comment:
@gpcastro I was thinking if we can get these changes in since they could help in perf investigation by us and as well as Jenny. Its a short gap solution.